src/WebJobs.Script/Utility.cs

```diff
@@ -520,8 +520,28 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static async Task MarkContainerDisabled()
         {
+            var disableContainerFilePath = Path.Combine(Path.GetTempPath(), ScriptConstants.DisableContainerFileName);
+            if (!FileUtility.FileExists(disableContainerFilePath))
+            {
+                await FileUtility.WriteAsync(disableContainerFilePath, string.Empty);
```

Comment:
Consider writing a value to the file, e.g. "This container instance is offline" or something to that effect

Reply:
Fixed.